druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -577,23 +577,22 @@ protected boolean enabled() {
    */
   @Test public void testCastToTimestamp() {
     final String sql = "select cast(\"timestamp\" as timestamp) from \"foodmart\"";
-    final String druidQuery = "{'queryType':'scan','dataSource':'foodmart',"
-        + "'intervals':['1900-01-09T00:00:00.000Z/2992-01-10T00:00:00.000Z']',"
-        + "'virtualColumns:[{'type':'expression','name':'vc',"
-        + "'expression':'timestamp_parse("
-        + "timestamp_format(\"__time\",'yyyy-MM-dd\\u0027T\\u0027HH:mm:ss.SSS\\u0027Z\\u0027',"
-        + "'America/New_York'),'yyyy-MM-dd\\u0027T\\u0027HH:mm:ss.SSS\\u0027Z\\u0027','UTC')',"
-        + "'outputType':'LONG'}],"
-        + "'columns':['vc'],"
-        + "'resultFormat':'compactedList'}";
+    final String druidQuery = "timestamp_format(\\\"__time\\\","
+        + "'yyyy-MM-dd\\\\u0027T\\\\u0027HH:mm:ss.SSS\\\\u0027Z\\\\u0027',"
+        + "'America/New_York'),'yyyy-MM-dd\\\\u0027T\\\\u0027HH:mm:ss.SSS\\\\u0027Z\\\\u0027','UTC')\"";
 
     CalciteAssert.that()
          .enable(enabled())
          .withModel(FOODMART)
          .with(CalciteConnectionProperty.TIME_ZONE.camelName(), "America/New_York")
          .query(sql)
          .runs()
-         .queryContains(druidChecker(druidQuery));
+         .queryContains(list -> {
+           assertThat(list.size(), is(1));
+           //remove the replace from druidChecker to preserve the single quotes
```

Comment:
Is the grammar correct? "replace" is a verb.

Reply:
I meant it as a call to the method replace()